en/lessons/basics/documentation.md

```diff
@@ -195,20 +195,21 @@ Examples
 ### Installing
 
 Assuming all is well and we're seeing the output above, we are now ready to set up ExDoc.
-In the `mix.exs` file, add the two required dependencies to get started: `:earmark` and `:ex_doc`.
+In the `mix.exs` file, add the `:ex_doc` dependency to get started.
 
 ```elixir
   def deps do
-    [{:earmark, "~> 1.2", only: :dev},
-    {:ex_doc, "~> 0.19", only: :dev}]
+    [{:ex_doc, "~> 0.21", only: :dev, runtime: false}]
   end
 ```
 
-We specify the `only: :dev` key-value pair as we don't want to download and compile these dependencies in a production environment.
-But why Earmark? Earmark is a Markdown parser for the Elixir programming language that ExDoc utilizes to turn our documentation within `@moduledoc` and `@doc` to beautiful looking HTML.
+We specify the `only: :dev` key-value pair as we don't want to download and compile the `ex_doc` dependencies in a production environment.
```

Comment:
``ex_doc` dependencies` -> ``ex_doc` dependency`

Reply:
Sure. I was referring to `ex_doc` and `earmark`, but I don't have a problem changing it either :)

Reply:
OK, change done 👍